workshops/management/commands/check_certificates.py

```diff
@@ -0,0 +1,56 @@
+import sys
+import os
+from django.core.exceptions import ObjectDoesNotExist
+from django.core.management.base import BaseCommand, CommandError
+from workshops.models import Award, Badge, Person
+
+
+class Command(BaseCommand):
+    args = '/path/to/certificates'
```

Comment:
It's waaay better to use `optparse` capabilities of Django. Take for example https://github.com/swcarpentry/amy/blob/develop/workshops/management/commands/upgrade_instructor_profiles.py#L75

You'd use `add_parameter` method and it would:

1. Give you a very nice help msg
2. Validate parameters for you